aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml

```
@@ -14,8 +14,8 @@
 
 apiVersion: v2
 name: kubernetes-dashboard
-version: 5.5.2
-appVersion: 2.5.1
+version: 5.5.3
```

```suggestion
version: 5.6.0
```
I am sorry, I did not realize earlier: this is new feature according to semver, could you bump minor version instead of patch?

---

done